dts/bindings/timer/andestech,machine-timer.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Andes Machine Timer
+
+  The Andes machine timer provides RISC-V prigileved mtime and mtimecmp
```

Comment:
prigileved -> privileged (here and in the rest of bindings)

Reply:
fixed on all bindings, thanks